<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Testing_window</class>
 <widget class="QMainWindow" name="Testing_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>671</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Testing_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="Legend_1"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Legend_2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Legend_3"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Legend_4"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Legend_5"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Legend_6"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Legend_7"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Legend_8"/>
          </item>
          <item>
           <widget class="QLineEdit" name="Legend_9"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="value_1"/>
          </item>
          <item>
           <widget class="QLineEdit" name="value_2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="value_3"/>
          </item>
          <item>
           <widget class="QLineEdit" name="value_4"/>
          </item>
          <item>
           <widget class="QLineEdit" name="value_5"/>
          </item>
          <item>
           <widget class="QLineEdit" name="value_6"/>
          </item>
          <item>
           <widget class="QLineEdit" name="value_7"/>
          </item>
          <item>
           <widget class="QLineEdit" name="value_8"/>
          </item>
          <item>
           <widget class="QLineEdit" name="value_9"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="unit_1"/>
          </item>
          <item>
           <widget class="QLineEdit" name="unit_2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="unit_3"/>
          </item>
          <item>
           <widget class="QLineEdit" name="unit_4"/>
          </item>
          <item>
           <widget class="QLineEdit" name="unit_5"/>
          </item>
          <item>
           <widget class="QLineEdit" name="unit_6"/>
          </item>
          <item>
           <widget class="QLineEdit" name="unit_7"/>
          </item>
          <item>
           <widget class="QLineEdit" name="unit_8"/>
          </item>
          <item>
           <widget class="QLineEdit" name="unit_9"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextEdit" name="textEdit_31">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Задание возбуждения&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="excitement_params"/>
          </item>
          <item>
           <widget class="QTextEdit" name="excitement_task">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="excitement_botton">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="text">
             <string>Задание управления</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QComboBox" name="management_params_1"/>
            </item>
            <item>
             <widget class="QTextEdit" name="management_task_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QComboBox" name="management_params_2"/>
            </item>
            <item>
             <widget class="QTextEdit" name="management_task_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="management_button">
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
